pkg/config/templates/rancherd-22-vm-import-controller.yaml

```diff
@@ -11,5 +11,6 @@ resources:
       enabled: false
       valuesContent: |
         image:
-          tag: v0.1.0
+          tag: v0.1.1
+          fullnameOverride: harvester-vm-import-controller
```

Comment:
Sorry, just found `fullnameOverride` should not be in `image`. It's on the top level.

Reply:
👍